quesma/telemetry/phone_home.go

```diff
@@ -539,6 +596,11 @@ func (a *agent) collect(ctx context.Context, reportType string) (stats PhoneHome
 	} else {
 		stats.ClickHouse = ClickHouseStats{Status: "paused"}
 	}
+	if !strings.HasPrefix(a.config.ClickHouse.ConnectorType, "hydrolix") { // we only check table sizes for ClickHouse
+		if tables, err := a.collectClickHouseTableSizes(ctx); err == nil {
+			logger.Info().Msgf("Usage report dababase=[%s] table sizes=%v", a.getDbInfo(), tables)
```

Comment:
Why is this logged only?  We can add this site to the clickhouse stats.

Reply:
+100 Would prefer to have that in telemetry report.